src/server/server.ts

```diff
@@ -1,50 +1,16 @@
-import express from "express";
-import cors from "cors";
-import path from 'path';
-import http from 'http';
-import dotenv from 'dotenv';
-import { Server } from "socket.io";
+import { app, envConfig } from './configs/root.config.js';
```

Comment:
```suggestion
import * as root from './configs/root.config.js';
```

just a suggestion .. if these start to get long you can change the import line to this.. and start using them like root.app

Reply:
I will keep this for now. Thanks!